Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
Yes - __CFRuntimeObjCClassTable is usable from other frameworks. If it weren't, we wouldn't be able to get it from Foundation, and the bridging tests we have so far for UICTFont shouldn't have worked.

---
In reply to: [79022634](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79022634) [](ancestors = 79022634,78866010,78862337)